data/Delimiter/tb_comment.yaml

```diff
@@ -0,0 +1,2 @@
+type: comment
+start: REM
```

Comment:
this deliminter should be called `rem.yaml`.  It is used by many, many early languages ;-)

Reply:
My bad, fixed!